lib/locales/es/frontend/alert-messages.json

```
@@ -10,7 +10,9 @@
   "withdrawal_amount_being_processed": "El retiro de {{amount}} está siendo procesado",
   "error_withdrawing_exists_pending_ones": "No se pueden realizar retiros mientras haya alguno pendiente.",
   "two_fa_code": {
+    "title": "Autenticación de Dos Factores",
     "required": "Es obligatorio tener activo el método de autenticación de dos factores",
+    "required_with_instruction_to_activate": "Es obligatorio tener el método de autenticación de dos factores activo. Para continuar, primero debes activarlo en la <1>página de configuración</1>.",
```

We can delete this `title` and `required_with_instruction_to_activate`, right?
```suggestion
    "required": "Es obligatorio tener activo el método de autenticación de dos factores",
```